libs/components/ag-grid/src/lib/modules/ag-grid/types/text-properties.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @internal
+ */
+export interface SkyTextProperties {
+  maxlength?: number;
+}
```

Comment:
☝️ What's the value of separating the `SkyNumberProperties` and `SkyTextProperties` from their `SkyCellEditorNumberParams` and `SkyCellEditorTextParams` interfaces? Feels like it's adding needless complexity. For example:

```
export interface SkyCellEditorNumberParams extends ICellEditorParams {
  skyComponentProperties?: { max?: number; min?: number; };
}
```

If we need to keep them separated for some reason, we should rename them to be more specific to AgGrid in case we need to export them to the public API in the future (`SkyNumberProperties` is a bit too generic).